### ISSUE TITLE
Sub-Domains: implement new get_domains method in winbind provider

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1070]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1066
- Created at 2011-10-27 10:29:02 by [sbose](https://pagure.io/user/sbose)
- Closed as Invalid
- Assigned to [sbose](https://pagure.io/user/sbose)
---

See https://fedorahosted.org/sssd/wiki/DesignDocs/SubDomains, depends on #1064

### Comments

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2011-10-27 15:50:29](https://pagure.io/SSSD/sssd/issue/1066#220439)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.8.0
owner: somebody => sbose
priority: major => blocker

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2011-12-10 18:36:06](https://pagure.io/SSSD/sssd/issue/1066#220440)**

Fields changed

milestone: SSSD 1.8.0 => NEEDS_TRIAGE

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2011-12-10 18:49:00](https://pagure.io/SSSD/sssd/issue/1066#220441)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.8 AD Integration NEEDS TRIAGE

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2011-12-12 16:25:06](https://pagure.io/SSSD/sssd/issue/1066#220442)**

Fields changed

milestone: SSSD 1.8 AD Integration NEEDS TRIAGE => SSSD Deferred

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2012-01-19 03:23:02](https://pagure.io/SSSD/sssd/issue/1066#220443)**

Fields changed

rhbz:  => 0

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2012-06-14 19:40:42](https://pagure.io/SSSD/sssd/issue/1066#220444)**

Fields changed

blockedby:  => 
blocking:  => 
feature_milestone:  => 
resolution:  => wontfix
status: new => closed

---

**Comment from [sbose](https://pagure.io/user/sbose) at [2017-02-24 14:21:56](https://pagure.io/SSSD/sssd/issue/1066#220445)**

**Metadata Update from @sbose**:
- Issue assigned to sbose
- Issue set to the milestone: SSSD Patches welcome
